core/services/ocr2/plugins/rebalancer/models/config.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+const PluginName = "liquidityRebalancer"
+
+type PluginConfig struct {
+	LiquidityManagerAddress Address   `json:"liquidityManagerAddress"`
+	LiquidityManagerNetwork NetworkID `json:"liquidityManagerNetwork"`
+	ClosePluginTimeoutSec   int       `json:"closePluginTimeoutSec"`
```

Comment:
It's more idiomatic Go to make durations `time.Duration`

```suggestion
	ClosePluginTimeout   time.Duration       `json:"closePluginTimeoutSec"`
```

Reply:
I leaned towards an `int` because that way our config will look like this:
```
cfg := `{"closePluginTimeoutSec": 120}`
```

instead of
```
cfg := `{"closePluginTimeout": 120000000000}`
```

which is `go` specific

Reply:
Hmm I'm not 100% sure but the plugin config in job specs is usually written as toml, e.g https://github.com/smartcontractkit/chainlink/blob/7f482a95c4de878e6a2a14c0311ab90ec23d7157/core/internal/features/ocr2/features_ocr2_test.go#L320-L336 or https://github.com/smartcontractkit/chainlink/blob/7f482a95c4de878e6a2a14c0311ab90ec23d7157/core/scripts/chaincli/handler/keeper_launch.go#L359-L363. So for this plugin you'd write it as

```toml
[pluginConfig]
closePluginTimeout = "120s"
```

i.e you'd just use the duration string representation. Go JSON unmarshalers will know how to unmarshal this format.

Reply:
I can't come up with a quick way to test this right now though but something to keep in mind for the future.